client/src/components/events/CreateEventTypeBookingForm.vue

```diff
@@ -260,8 +260,11 @@ export default {
             try {
                 if (this.$refs.form.validate()) {
                     this.prepareData();
-                    await this.addEventType(this.data);
-                    this.$router.push({ name: 'EventTypes' });
+                    const response = await this.addEventType(this.data);
+                    this.$router.push({
+                        path:
+                            'new-event-type-options?eventTypeId=' + response.id
```

Comment:
Use the name of the variables from the route, so that when you change the path in routes in the code, it automatically changes